test/helper/cleaner.js

```diff
@@ -0,0 +1,9 @@
+const { client } = require('../../app/db')
```

Comment:
This should be in a `seeds/projects.js`.
Have a look at the repo you created before to see how it should be implemented.